### PR TITLE
feat(Breadcrumb): breadcrumbs combined with page title

### PR DESCRIPTION
**Pull request by amirfefer**
Adds breadcrumbs combined with page title as it described in the patternfly design page. 
Storybook:
https://rawgit.com/amirfefer/patternfly-react/storybook/feature/breadcrumbs-combined-with-title/index.html?selectedKind=About%20Modal&selectedStory=AboutModal&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs

### CODE REVIEW

File: src/components/Breadcrumb/Breadcrumb.js [hunk 1]
```diff
@@ -1 +1,19 @@
-export { Breadcrumb as default } from 'react-bootstrap';
+import React from 'react';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import { Breadcrumb as PfBreadcrumb } from 'react-bootstrap';
```
Comment (jeff-phillips-18) [inline src/components/Breadcrumb/Breadcrumb.js]: bsBreadCrumb would be more accurate.

File: sass/patternfly-react/_patternfly-react.scss [hunk 2]
```diff
@@ -1,3 +1,7 @@
 /**
   Patternfly React Partials
 */
+.breadcrumbs-title .active span {
```
Comment (jeff-phillips-18) [inline sass/patternfly-react/_patternfly-react.scss]: Should qualify this as a PF style:  .breadcrumbs-pf-title

**Comment by sharvit** [comment]
Should the css part of it be apart of patternfly-core?

I the documentation there is very cool design for breadcrumbs and breadcrumbs-as-title
http://www.patternfly.org/pattern-library/navigation/breadcrumbs/#design

But there is no code example about how to use it,
**do we have a class that makes this magic happen or it is still in development?**

**Comment by amirfefer** [comment]
@jeff-phillips-18 Thanks for your review.
Should the css class implemented [here](https://github.com/patternfly/patternfly/blob/64357c539ed7f8723f5dfe5b2c8680f869246cf6/src/less/breadcrumbs.less) ?

**Comment by jeff-phillips-18** [comment]
@amirfefer Yes, it should be implemented in PF core (you have the correct location). If this is pressing we can add it to this repo in the less and scss directories and file an issue against PF core.

File: sass/patternfly-react/_patternfly-react.scss [hunk 3]
```diff
@@ -1,3 +1,7 @@
 /**
   Patternfly React Partials
 */
+.breadcrumbs-pf-title .active span {
+  font-size: 16px;
+  vertical-align: middle;
+}
```
Comment (jeff-phillips-18) [inline sass/patternfly-react/_patternfly-react.scss]: Sorry, updates to the [CONTRIBUTING.MD](https://github.com/patternfly/patternfly-react/blob/master/CONTRIBUTING.md) file had not yet been merged. It has been merged now. Please see the  [Adding Styling for your Components](https://github.com/patternfly/patternfly-react/blob/master/CONTRIBUTING.md#adding-styling-for-your-components) section.

**Comment by amirfefer** [comment]
@jeff-phillips-18 Thanks, and yes it's quite pressing.
I've followed `Adding Styling for your Components` section.

**Comment by amirfefer** [comment]
@Rohoover, Any thoughts ? 
https://rawgit.com/amirfefer/patternfly-react/storybook/feature/breadcrumbs-combined-with-title/index.html?selectedKind=About%20Modal&selectedStory=AboutModal&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs

**Comment by amirfefer** [comment]
@serenamarie125 
Thanks for your comment,
it looks like this double `>` is implemented in [patternfly-core](https://github.com/patternfly/patternfly/blob/64357c539ed7f8723f5dfe5b2c8680f869246cf6/src/less/breadcrumbs.less#L14)

**Comment by amirfefer** [comment]
@Rohoover 
Could you choose a better position for this active breadcrumb? 
breadcrumb-noallignn

breadcrumb-1

breadcrumb-2

**Comment by LHinson** [comment]
@amirfefer that's right, a bug has been filed in pf-core https://github.com/patternfly/patternfly/issues/949 and we'll be making this change to a single > in an upcoming release.

**Comment by amirfefer** [comment]
Thanks @Rohoover 
Done :)
[updated storybook](https://rawgit.com/amirfefer/patternfly-react/storybook/feature/breadcrumbs-combined-with-title/index.html?selectedKind=Breadcrumb&selectedStory=Breadcrumb%20combined%20with%20page%20title&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs)

**Comment by jeff-phillips-18** [comment]
@serenamarie125 Any further issues/concerns?

**Pull request merged**